conf/dev_base.conf

```diff
@@ -18,6 +18,7 @@ features {
   useWhitelistOnRepurpose = true
   keepSomeMetaOnRepurpose = [ attr1, attr2 ]
   deleteSomeMetaOnRepurpose = [ attr4, attr3 ]
+  syslogAsset = tumblrtag1
```

Comment:
Well, I needed this to be able to see errors I was getting.  so it is technically general cleanup, but since it was a one liner, I decided to sneak it in.